conn/config.c

```diff
@@ -131,7 +131,7 @@ struct ConfigDef ConnVars[] = {
     "(ssl) Use TLSv1.3 for authentication"
   },
 #ifdef USE_SSL_OPENSSL
-  { "ssl_usesystemcerts", DT_BOOL, &C_SslUsesystemcerts, true, 0, NULL,
+  { "ssl_use_system_certs", DT_BOOL, &C_SslUseSystemCerts, true, 0, NULL,
```

Comment:
You've renamed the global variable here: `C_SslUseSystemCerts`, but it exists in a few other places!
Build `./configure --ssl` fails.

Reply:
Done!